### ISSUE TITLE
Error while updating property 'source' of a view managed by: webimageview

### ISSUE THREAD

**Issue by arjdev69** [vovkasm/react-native-web-image#21]
![WhatsApp Image 2019-04-20 at 03 18 12](https://user-images.githubusercontent.com/6216736/56453628-8fd6da00-631b-11e9-8936-b9c5b69b809c.jpeg)

**Comment by vovkasm** [comment]
Can you create example of crash in `samples/SampleApp`?

Note: to run sample on android, do the following:
1. cd <project directory>
2. `npm install` (or `npm ci`)
3. `npm start`
4. `npm run run-sample-android` (or import and run project from Android Studio)

**Comment by vbanurag** [comment]
same issue with me as well.

But I'm using glide version 4.9 at native side and in react native using this lib. So when I'm compiled the code it gives me same issue even the version is also same.

**Comment by vovkasm** [comment]
If this issue exists, it should not depend on glide version. But it will be very useful if someone interested in resolution will create test or at least reproducible example in `samples/SampleApp` project.

**Comment by afeichuanqi** [comment]
Has this problem been solved?

**Comment by vovkasm** [comment]
@afeichuanqi No, and I still hope that one interested in the issue will contribute at least test case.
Personally I'm successfully use this module in production, but my use case don't change source on the fly, so this issue not in priority.
